### ISSUE TITLE
Create public website for Nodeschool Bratislava

### ISSUE THREAD

**Issue by yangwao** [nodeschool/bratislava#17]
Ahoj, druhy nodeschool za nami a patrilo by sa mat aj stranku. Kedze podla [pyramidy](https://github.com/nodeschool/organizers/wiki) by to bolo vhodnejsie pre sirenie a davanie ludom viac vediet, kde a ako sa mozu naucit programovat :)
Na inspiraciu davam [list Nodeschoolov po svete ktore uz maju stranku](http://nodeschool.io/chapters.html)
Z nich vyberam napr 
- https://github.com/nodeschool/organizers/wiki
- http://nodeschool.io/vienna/
- http://nodeschool.io/bristol/
- http://nodeschool.io/stuttgart/
- http://nodeschool.io/istanbul/ 
- http://nodeschool.io/toronto/

Kto sa podujme? Myslim ze na Slovensku su ludia, ktori by sa chceli ucit. Mozem prilozit ruku k dielu a pomoct, ked budem vediet.

**Comment by adammockor** [comment]
Ak by niekto dodal dizajn, som ochotny programovat.

Popripade ak by niekto velmi tuzil vyskusat si nejake node.js CMS, http://keystonejs.com/ je velmi zaujmavy projekt.

**Comment by yangwao** [comment]
@adammockor chcel by som to nechat super simple, len vymenim link na typeform, commitnem a je.
Cize uz len niekto komu by sa chcel robit nejaky dizajn.
Malo by to obsahovat 
- [ ] embed ti.to on web
- [x]  link na typeform
- [x]  ako sa dostat na miesto
- [ ]  co si nezabudnut prejst pre newcomers
- [x]  odkaz na gitter
- [ ]  ako prebieha taky nodeschool (selfguided)
- [x] link na code of conduct
- [ ]  a veci co som zabudol z [organizers wiki](https://github.com/nodeschool/organizers/wiki), ktore si myslite, ze by nemali chybat vytiahnute

**Comment by yangwao** [comment]
Ok, vidim ze sa nikto nema, tak hned ako sa zaucim http://gohugo.io/ tak neico vygenerujem :) 
Je aj v nodejs, ale nedocital som to http://metalsmith.io/ :)

**Comment by yangwao** [comment]
este teda sponsors sekcia
- [x] - past sponsors s countom

**Comment by yangwao** [comment]
a toto este nejak zakomponujeme https://github.com/nodeschool/bratislava/issues/6

**Comment by yangwao** [comment]
http://nodeschool.io/bratislava/
- [ ]  zmena pisma, asi na roboto 2.0 ?
- [ ]  datum mozno normalnym weightom ( font-size: 1rem; font-weight: 300; ~)
- [ ] button/y Give a talk / Request Topic na => https://github.com/nodeschool/bratislava/blob/master/talks.md
- [ ]  mapa zarovno s how to start sekciou dolu 
- [ ]  mapa heigh 400px vyzera lepsie asi, vidiet dunaj :dancers: 
- [ ]  footer linkov asi center?
- [ ]  na linuxe sa nasiel nejaky weird bug pri hoveri buttonov 
- [ ]  link na toto https://gitter.im/nodeschool/discussions
- [ ]  za seba musim este updatnut text :)
- [ ]  popis ze nodeschool je selfguided
- [ ]  https://github.com/nodeschool/bratislava/issues/6 
  ![pasted image at 2016_03_07 01_15 pm](https://cloud.githubusercontent.com/assets/5887929/13569518/c1111854-e468-11e5-869c-a11b18cb41e9.png)

**Inak perfect! :rocket: :)**

**Comment by yangwao** [comment]
Asi ten popis ze nodeschool je selfguided a blabla tieto reci este by som nemal zabudnut za seba.

**Comment by yangwao** [comment]
mozno este link na toto https://gitter.im/nodeschool/discussions

**Comment by mtakac** [comment]
Ahoj, 

na http://nodeschool.io/ je datum najblizsieho meetupu `4.2.2016 9:30AM, FB` event ma `Saturday, April 9 at 9:30 AM - 4 PM` a https://yangwao.typeform.com/to/wdZuNE `Saturday, February 27th, 2016 - 9:00 AM - 16:00 GMT+1`.

**Comment by yangwao** [comment]
to je logicka hadanka! tento vikend to idem zosumarizovat, smerodatny je fb
event a github issue https://github.com/nodeschool/bratislava/issues/24 tj
9.4. (aj ked som to chcel posuvat ale je potom makerfaire Vienna tusim)
ospravedlnujem sa ale je to skvela prilezitost mi pomoct orgovat! :)
vdaka za hint

**Comment by mtakac** [comment]
@yangwao fuu no pomoc s orgovanim... ako rad by som, ale momentalne som na tom nie uplne dobre s casom a nerad by som nasluboval nieco co potom nebudem vediet dodrzat. Priebezne to tu sledujem a ak sa najde nieco pricom budem mat pocit, ze by som to stihal a vedel pomoct tak sa rad zapojim.

Good luck!

**Comment by kasicka** [comment]
Nejaky plan updatnut https://nodeschool.io/bratislava/ ?

**Comment by yangwao** [comment]
@kasicka áno, dnes zosumarizujem popis a rozvediem description, nech je to aktuálne, vďaka za notice :)
Aktualne je to v takom semi stave, ale vzdy sa da spravit lepsie, kludne posli PR :)
